src/test/java/nextstep/ladder/LineTest.java

```diff
@@ -0,0 +1,31 @@
+package nextstep.ladder;
+
+import nextstep.ladder.domain.Line;
+import nextstep.ladder.domain.Point;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class LineTest {
+
+    @Test
+    @DisplayName("라인이 겹칠 때 생성 불가 테스트")
+    void lineOverLabTest() {
+        assertThatThrownBy(() -> new Line(List.of(new Point(true), new Point(true), new Point(false))))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining("중복");
+    }
+
+    @Test
+    @DisplayName("라인 생성 테스트")
+    void lineConstructTest() {
+        Line line = new Line(List.of(new Point(false),new Point(true),new Point(false)));
```

Comment:
```suggestion
        Line line = new Line(List.of(false,true,false));
```

생성자를 다양하게 사용할수 없을까요?

Reply:
이전에 `List<Bollean> points` 매개변수를 가지는 생성자를 만들었을 때, 제네릭 오버로딩 문제로 따로 만들 수 없었네요,,
다른 방법이 있을까요?